add_definitions (#223)
jaybee84/ml-in-rd: content/10. box_2_definitions.md

Regularization is an approach to reduce overfitting of models to training data, where a penalty or constraint is added to a model trained on a training dataset to avoid making large prediction errors on the evaluation dataset.

### Knowledge Graph
A knowledge graph is a network representation of human knowledge about a domain, abstracted into nodes and edges. 
Any entity of interest (for example a gene, a disease, a protein, or a cell-line) can be represented as a node in a knowledge graph.
All nodes can be linked through edges that represent known relationships between the nodes. 
Edges can be directed, indicating that the order of the nodes is important for encoding the relationship, or undirected.
For example, a gene (node) can be linked to a protein (node) using a directed edge that represents the relationship that the protein is generated through the transcription and translation of the gene. 

> Knowledge graphs serve to integrate data that exist is distributed sources, encode human readable knowledge in machine readable format, and evolve in a flexible manner to integrate new knowledge as it becomes available.

Author: ```suggestion
Knowledge graphs serve to integrate data that exist in distributed sources, encode human readable knowledge in machine readable format, and evolve in a flexible manner to integrate new knowledge as it becomes available.
```

Regularization is an approach to reduce overfitting of models to training data, where a penalty or constraint is added to a model trained on a training dataset to avoid making large prediction errors on the evaluation dataset.

### Knowledge Graph

> Human knowledge about a domain can be abstracted into a form of network of nodes and edges collectively forming a knowledge graph. In this network any entity of interest (for example a gene, a disease, a protein, or a cell-line) can be represented as a node. All nodes can be linked in a directed manner through edges that represent known relationships between the nodes. For example a gene (node) can be linked to a protein (node) using a directed edge that represents the relationship that the protein is generated through the transcription and translation of the gene. Knowledge graphs serve to integrate data that exist is distributed sources, encode human readable knowledge in machine readable format, and evolve in a flexible manner to integrate new knowledge as it becomes available.

Reviewer: One sentence per line, and some other suggestions too. Don't love what I wrote about directed edges.

```suggestion
A knowledge graph is a network representation of human knowledge about a domain, abstracted into nodes and edges. 
Any entity of interest (for example a gene, a disease, a protein, or a cell-line) can be represented as a node in a knowledge graph.
All nodes can be linked through edges that represent known relationships between the nodes. 
Edges can be directed, indicating that the order of the nodes is important for encoding the relationship, or undirected.
For example, a gene (node) can be linked to a protein (node) using a directed edge that represents the relationship that the protein is generated through the transcription and translation of the gene. 
Knowledge graphs serve to integrate data that exist is distributed sources, encode human readable knowledge in machine readable format, and evolve in a flexible manner to integrate new knowledge as it becomes available.
```

### Rare Disease

> According to the Orphan Drug Act[@https://www.fda.gov/media/99546/download] of United States of America, diseases or conditions that impact less than 200,000 people in the U.S are considered to be rare diseases. European Union defines a disease as rare when it affects less than 1 in 2,000 people.

Reviewer: ```suggestion
According to the Orphan Drug Act[@https://www.fda.gov/media/99546/download] of United States of America, diseases or conditions that impact less than 200,000 people in the U.S are considered to be rare diseases. 
The European Union defines a disease as rare when it affects less than 1 in 2,000 people.
```
